src/test/resources/application.properties

```diff
@@ -4,6 +4,11 @@ workspace.maxStairwayThreads=4
 workspace.resourceId=mc-terra-workspace-manager
 workspace.stairwayTimeoutSeconds=1800
 
+spring.cloud.gcp.trace.enabled=true
```

Comment:
Do we want to enable tracing for all tests? It seems like unit tests would pollute the dev environments logs when we could just turn it off

Reply:
true! I will disable for tests